runtime/src/compute_budget_details.rs

```diff
@@ -8,46 +8,46 @@ use {
 };
 
 #[derive(Clone, Debug, PartialEq, Eq)]
-pub struct TransactionPriorityDetails {
-    pub priority: u64,
+pub struct ComputeBudgetDetails {
```

Comment:
at first glance, I wanted to nit pick the name `ComputeBudgetDetails` here, cause it implies to details of `ComputeBudget` defined `program-runtime/src/compute_budget.rs`. But then, this file is on its way out, we don't needed it when start to using runtime-transaction. I'm OK with this change for now.